frontend/packages/dev-console/src/components/guided-tour/index.ts

```diff
@@ -10,44 +11,50 @@ const getSelector = (id: string): string => `[data-tour-id="${id}"]`;
 
 export const getGuidedTour = (): TourDataType => ({
   intro: {
-    heading: 'Welcome to Dev Perspective!',
+    heading: i18n.t('devconsole~Welcome to Dev Perspective!'),
```

Comment:
We should be avoiding the use of the global `i18n.t`
You can pass in a translation function.